platform/server-bundle/src/main/resources/webapp/js/controllers.js

```diff
@@ -528,6 +530,11 @@
 
         $scope.getStartupViewData = function() {
             $scope.doAJAXHttpRequest('GET', '../server/startupviewdata', function (data) {
+                if(data.languages['zh_TW.Big5'] !== undefined) { delete data.languages['zh_TW.Big5']; }
```

Comment:
Let's style this code properly. The content of the if, as well as the closing brace should be in a new line. Moreover, let's add a comment with the ticket number, explaining that we temporarily hide this, until it's fixed.